source/game/field/obj/ObjectWoodbox.hh

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "game/field/obj/ObjectBreakable.hh"
+
+namespace Field {
+
+class ObjectWoodbox : public ObjectBreakable {
+public:
+    ObjectWoodbox(const System::MapdataGeoObj &params);
+    ~ObjectWoodbox() override;
+
+    void init() override;
+    void enableCollision() override;
+    void onRespawn() override;
```

Comment:
These should be moved down below `calcCollisionTransform`. We prefer to declare things in-order from base class to derived class. So the `ObjectBase` overrides (`init()`, `getKclName`, `calcCollisionTransform`) should occur first.